Realm/RLMSchema_Private.h

```diff
@@ -32,19 +35,21 @@ extern "C" {
 @property (nonatomic, readwrite, copy) NSArray *objectSchema;
 
 // schema based on runtime objects
-+(instancetype)sharedSchema;
++ (instancetype)sharedSchema;
```

Comment:
sharedSchema returns nil during initialization of the shared schema (and do we rely on that in `-[RLMObjectBase init]`).